packages/driver/src/cypress/mocha.ts

```diff
@@ -490,7 +458,28 @@ function createCalculateTestStatus (Cypress: Cypress.Cypress) {
   Test.prototype.calculateTestStatus = function () {
     const retriesConfig = getNormalizedRetriesConfig(Cypress)
 
-    return calculateTestStatus(this, retriesConfig)
+    //@ts-expect-error
+    const actions = Cypress.__actions
+
+    const action = actions?.find((a) => a.clientId === this.id && a.action === 'BURN_IN') ?? {}
+
+    const latestScore = action?.payload?.startingScore ?? -2
```

Comment:
can we make the change we talked about to?
`const latestScore = action?.payload?.startingScore === undefined ? -2 : action?.payload?.startingScore`